src/hotspot/share/opto/divnode.cpp

```diff
@@ -1260,6 +1337,28 @@ const Type* ModDNode::Value(PhaseGVN* phase) const {
   return TypeD::make(jdouble_cast(xr));
 }
 
+//=============================================================================
+//------------------------------UModINode---------------------------------------
+// Unsigned integer modulus
+  const Type* UModINode::Value(PhaseGVN* phase) const {
+    return TypeInt::INT;
+  }
+
+  Node* UModINode::Ideal(PhaseGVN *phase, bool can_reshape) {
+    return NULL;
+  }
+
```

Comment:
Please remove Value and Ideal routines if no explicit transforms are being done.

Reply:
Will remove the unused transformations.